__tests__/app.test.js

```diff
@@ -70,3 +70,49 @@ describe("GET-/api/reviews", () => {
       });
   });
 });
+
+describe.only("/api/reviews/:review_id", () => {
+  test("GET - 200: Returns the review with the correct ID", () => {
+    return request(app)
+      .get("/api/reviews/3")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body).toEqual({
+          review_id: 3,
+          title: "Ultimate Werewolf",
+          review_body: "We couldn't find the werewolf!",
+          designer: "Akihisa Okui",
+          review_img_url:
+            "https://www.golenbock.com/wp-content/uploads/2015/01/placeholder-user.png",
+          votes: 5,
+          category: "social deduction",
+          owner: "bainesface",
+          created_at: "2021-01-18T10:01:41.251Z",
+        });
+      });
+  });
+  test("GET - 404: Route not found", () => {
+    return request(app)
+      .get("/api/review/2")
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Route not found");
+      });
+  });
+  test("GET - 400: Bad request", () => {
+    return request(app)
+      .get("/api/reviews/dog")
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Bad request");
+      });
+  });
+  test("GET - 400: Good request but id does not exist", () => {
+    return request(app)
+      .get("/api/reviews/9999")
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Invalid Id");

```

Comment:
Invalid ID would usually suggest an ID that does not conform to the data type expected. A message like 'ID not found' would be more useful, but just a very minor, optional suggestion.